documentstore_migracao/export/sps_package.py

```diff
@@ -280,24 +280,21 @@ def asset_name(self, img_filename):
 
     @property
     def assets(self):
-        xpaths = (
-            '//graphic[@xlink:href]',
-            '//media[@xlink:href]',
-            '//inline-graphic[@xlink:href]',
-            '//supplementary-material[@xlink:href]',
-            '//inline-supplementary-material[@xlink:href]',
+        xpath = (
```

Comment:
Quando implementei essa função originalmente eu medi o tempo de execução com as 2 abordagens: usando a avaliação sucessiva de expressões e uma única usando o conetivo _ou_. A primeira se mostrou significativamente mais rápida (não me lembro agora dos valores), por isso ela foi escolhida. Há algum motivo para a mudança?

Reply:
@gustavofonseca, lembro que você também usava o itertools.chain(). Fiz a mudança para retirar a necessidade de 2 `for`s e o retorno dos assets na ordem em que aparecem, mas confesso não ter me atentado especificamente a questão da performance.

Reply:
> @gustavofonseca, lembro que você também usava o itertools.chain(). Fiz a mudança para retirar a necessidade de 2 fors

Exato, que curiosamente tinham um desempenho melhor do que da outra forma. Enfim, se estivessemos com mais tempo eu iria sugerir que uma comparação fosse feita, mas não temos tempo para isso. 

Fica como curiosidade =]

Reply:
@gustavofonseca, fiz a alteração usando o itertools.